src/resources/postcss/tribe-common-admin.pcss

```
@@ -1014,6 +1014,17 @@ a.tribe-rating-link {
 	display: none;
 }
 
+/* = Widget Admin Screen
+============================ */
+.tribe-events-widget-admin-form__input-section p {
+	margin: 0;
+}
+
+.tribe-events-widget-admin-form__input-section h4 {
+	margin: .5em 0;
+}
```

Not sure if this is how we prefer that sort of thing or if we want them separated

```suggestion
.tribe-events-widget-admin-form__input-section {
	h4 {
		margin: .5em 0;
	}

	p {
		margin: 0;
	}
}
```

---

Yes - this is how we do it 🎵 
expand below down to https://github.com/moderntribe/tribe-common/pull/1468/files#diff-66301efaf5cd9fe690941a9f2a1d29d09d6c9083b15ad2ef197dc528c07cf9ffR1042 and you'll see an example ( `.events-cal .subsubsub`, `.events-cal .search-box`, etc)